dub.json

```diff
@@ -23,7 +23,7 @@
   "subPackages": [
```

Comment:
In the `xml` branch, "subPackages" is no longer relevant: remove it?

Reply:
I also listed importPaths explicitely as dub provides the wrong import path otherwise.